workflows/prognostic_c48_run/runtime/radiation_wrapper.py

```diff
@@ -0,0 +1,484 @@
+        rad_config["indcld"] = default_rad_config["indcld"]
+        rad_config["num_p3d"] = default_rad_config["num_p3d"]
+        rad_config["npdf3d"] = default_rad_config["npdf3d"]
+        rad_config["ncnvcld3d"] = default_rad_config["ncnvcld3d"]
+        rad_config["lmfdeep2"] = default_rad_config["lmfdeep2"]
+        rad_config["lmfshal"] = default_rad_config["lmfshal"]
+        rad_config["sup"] = default_rad_config["sup"]
+        rad_config["kdt"] = default_rad_config["kdt"]
+        rad_config["do_sfcperts"] = default_rad_config["do_sfcperts"]
+        rad_config["pertalb"] = default_rad_config["pertalb"]
+        rad_config["do_only_clearsky_rad"] = default_rad_config["do_only_clearsky_rad"]
+        rad_config["swhtr"] = physics_namelist["swhtr"]
+        rad_config["solcon"] = default_rad_config["solcon"]
+        rad_config["lprnt"] = default_rad_config["lprnt"]
+        rad_config["lwhtr"] = physics_namelist["lwhtr"]
+        rad_config["lssav"] = default_rad_config["lssav"]
+
+        return rad_config
+
+    def rad_update(
+        self,
+        time: cftime.DatetimeJulian,
+        dt_atmos: float,
+        forcing_dir: str = "./data/forcing",
+    ) -> None:
+        """Update the radiation driver's time-varying parameters"""
+        if self._rad_config is None:
+            raise ValueError(
+                "Radiation driver not initialized. `.rad_init` must be called "
+                "before `.rad_update`."
+            )
+        # idat is supposed to be model initalization time but is unused w/ current flags
+        idat = np.array(
+            [time.year, time.month, time.day, 0, time.hour, time.minute, time.second, 0]
+        )
+        jdat = np.array(
+            [time.year, time.month, time.day, 0, time.hour, time.minute, time.second, 0]
+        )
+        fhswr = np.array(float(self._rad_config["fhswr"]))
+        dt_atmos = np.array(float(dt_atmos))
+        aerosol_data = getdata.aerosol(forcing_dir)
+        _, solar_data = getdata.astronomy(forcing_dir, self._rad_config["isolar"])
+        gas_data = getdata.gases(forcing_dir, self._rad_config["ictmflg"])
+        self._driver.radupdate(
+            idat,
+            jdat,
+            fhswr,
+            dt_atmos,
+            self._rad_config["lsswr"],
+            aerosol_data["kprfg"],
+            aerosol_data["idxcg"],
+            aerosol_data["cmixg"],
+            aerosol_data["denng"],
+            aerosol_data["cline"],
+            solar_data,
+            gas_data,
+        )
+
+    def rad_compute(
+        self,
+        state: State,
+        tracer_metadata: Mapping[str, Any],
+        time: cftime.DatetimeJulian,
+        dt_atmos: float,
+        rank: int,
+        lookup_dir: str = "./data/lookup",
+    ):
+        if self._rad_config is None:
+            raise ValueError(
+                "Radiation driver not initialized. `.rad_init` must be called "
+                "before `.rad_compute`."
```

Comment:
Maybe just call `rad_init` from the constructor and remove this error message. Is there some use case for the lazy loading?
```suggestion
```